src/libtiled/mapreader.cpp

```diff
@@ -1349,7 +1361,6 @@ void MapReaderPrivate::readProperty(Properties *properties)
     properties->insert(propertyName, variant);
 }
 
-
```

Comment:
There's two empty lines here because the definitions below are for a different class, so please don't remove one of them. :-)